### PR TITLE
Section 1

### PR DESCRIPTION
**Pull request by ArturMay**
@RuslanFahrenheit, HTML and CSS for the first section have been added

### CODE REVIEW

File: scss/style.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
+
```
Comment (RuslanFahrenheit) [inline scss/style.scss]: Redundant empty line.

File: scss/style.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+h1 {
+    font-size: 54px;
+    color: #fff;
+    margin-bottom: 40px;
+}
+
+.highlighter {
+    font-size: 90px;
+    padding:0 20px;
```
Comment (RuslanFahrenheit) [inline scss/style.scss]: Code style. Check formatting all over the file.

File: scss/style.scss [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    background-size: cover;
+    text-align: center;
+    min-height: 900px;
+}
+
+.container {
+	max-width: 1515px;
+	margin: 0 auto;
+	padding: 0 15px;
+    // border: 1px solid rgb(69, 151, 80);
```
Comment (RuslanFahrenheit) [inline scss/style.scss]: Remove commented code.

File: scss/style.scss [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            color: red;
+        }
+    }
+}
+
+.company-slogan {
+    color: #fff;
+    font-size: 20px;
+    margin-bottom: 220px;
+}
```
Comment (RuslanFahrenheit) [inline scss/style.scss]: Add a blank line at the end.

File: index.html [hunk 5]
```diff
@@ -5,9 +5,32 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>CSS Layouts</title>
+  <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-    <!-- Your code goes here -->
-    <!-- Your code goes here -->
+    <header>
```
Comment (RuslanFahrenheit) [inline index.html:12]: An entire section is wrapped in `<header>`. Why? Is this really header?

File: index.html [hunk 6]
```diff
@@ -5,9 +5,32 @@
...
+      <div class="visual-box">
```
Comment (RuslanFahrenheit) [inline index.html]: What is the meaning of 'visual-box' class name? I think there are more semantically correct options.
E.g. https://www.google.com/search?client=firefox-b-d&q=hero+section

File: index.html [hunk 7]
```diff
@@ -5,9 +5,32 @@
...
+        <div class="container">
+          <div class="header-wrap">
+            <div class="logo">
+              <a href="#">BLACK & WHITE</a>
+            </div>
```
Comment (RuslanFahrenheit) [inline index.html]: It's a bad idea to use text as a logo. It should always looks the same. Imagine fonts not loading. Always use image as a logo.
E.g. https://www.adidas.com/us 
They don't type 'ADIDAS'. It's an image.

File: index.html [hunk 8]
```diff
@@ -5,9 +5,32 @@
...
+          <h1>BLACK<span class="highlighter">&</span>WHITE</h1> 
+          <p class="company-slogan">An Original Brand Since 1965</p>
+          <a href="#" class="btn">
+            <span>Show More</span>
+          </a>
+        </div>
+      </div>
+    </header>
 </body>
 </html>
```
Comment (RuslanFahrenheit) [inline index.html]: Add blank line.

File: index.html [hunk 9]
```diff
@@ -5,9 +5,32 @@
...
+                  <li><a href="#">About</a></li>
+                  <li><a href="#">Vacancies</a></li>
+                  <li><a href="#">Shop</a></li>
+              </ul>
+            </nav>
+          </div> 
+          <h1>BLACK<span class="highlighter">&</span>WHITE</h1> 
+          <p class="company-slogan">An Original Brand Since 1965</p>
+          <a href="#" class="btn">
+            <span>Show More</span>
```
Comment (RuslanFahrenheit) [inline index.html]: What is the purpose of `span` inside the link tag?

File: index.html [hunk 10]
```diff
@@ -5,9 +5,32 @@
...
+            <nav class="nav">
+              <ul>
+                  <li><a href="#">Home</a></li>
+                  <li><a href="#">About</a></li>
+                  <li><a href="#">Vacancies</a></li>
+                  <li><a href="#">Shop</a></li>
+              </ul>
+            </nav>
+          </div> 
+          <h1>BLACK<span class="highlighter">&</span>WHITE</h1>
```
Comment (RuslanFahrenheit) [inline index.html]: I suppose "highlighting" is more about making something stand out with color.
Not critical, but think about more semantically correct name.

**Review by RuslanFahrenheit — CHANGES_REQUESTED** [review]
1. Why do you have two identical commits: "Added HTML and CSS for the first section"? First commit is about adding configuration for processor. Describe your commits carefully.
Also read [Как писать сообщения коммитов в Git](https://medium.com/grisme/%D0%BA%D0%B0%D0%BA-%D0%BF%D0%B8%D1%81%D0%B0%D1%82%D1%8C-%D1%81%D0%BE%D0%BE%D0%B1%D1%89%D0%B5%D0%BD%D0%B8%D1%8F-%D0%BA%D0%BE%D0%BC%D0%BC%D0%B8%D1%82%D0%BE%D0%B2-%D0%B2-git-9ed19ebc5ebf).
2. I don't see typography styles.
3. The section should be full-height on all screens. Don't use big margin values for that. 
4. Don't write styles in one file. Read:
- https://qna.habr.com/q/741337
- http://prgssr.ru/development/estetichnyj-sass-chast-1-arhitektura-i-organizaciya.html#heading-section

File: index.html (hunk 9 above)
Comment (ArturMay) [reply index.html]: For making bigger font-size to the symbol "&"

File: scss/style.scss (hunk 2 above)
Comment (ArturMay) [reply scss/style.scss]: +checked

File: scss/style.scss (hunk 3 above)
Comment (ArturMay) [reply scss/style.scss]: I will remove it in the end, it helps me

File: scss/style.scss (hunk 4 above)
Comment (ArturMay) [reply scss/style.scss]: done

File: index.html (hunk 5 above)
Comment (ArturMay) [reply index.html:12]: main has been added. Now header contains logo and menu (navigation) only

File: index.html (hunk 6 above)
Comment (ArturMay) [reply index.html]: 'visual box has been changed to 'hero-section'.
Also, now the hero-section has a min-height of 100vh and big margins have been removed.

File: index.html (hunk 7 above)
Comment (ArturMay) [reply index.html]: Text in the logo has been changed to image

File: index.html (hunk 8 above)
Comment (ArturMay) [reply index.html]: done. it was the hardest one :)

File: scss/style.scss (hunk 1 above)
Comment (ArturMay) [reply scss/style.scss]: done

File: index.html (hunk 10 above)
Comment (ArturMay) [reply index.html]: Changed to "enlarger"

**Comment by RuslanFahrenheit** [comment]
TODO:
- style.scss - only imports here;
- code style;
- remove %button extender;
- add class to `<header>`.

**Pull request merged**